pydoctor/sphinx_ext/build_apidocs.py

```diff
@@ -35,7 +43,37 @@ def on_build_finished(app: Sphinx, exception: Exception) -> None:
     """
     Called when Sphinx build is done.
     """
-    config: Config = app.config  # type: ignore[has-type]
+    runs = app.config.pydoctor_args
+    placeholders = {
+        'outdir': app.outdir,
+        }
+
+    if not isinstance(runs, Mapping):
+        # We have a single pydoctor call
+        runs = {'main': runs}
+
+    for key, value in runs.items():
+        arguments = _get_arguments(value, placeholders)
+
+        options, _ = parse_args(arguments)
+        output_path = pathlib.Path(options.htmloutput)
+        sphinx_files = pathlib.Path(options.htmloutput + '_sphinx_files')
+
+        temp_path = pathlib.Path(options.htmloutput + '_pydoctor_temp')
+        shutil.rmtree(sphinx_files, ignore_errors=True)
+        output_path.rename(sphinx_files)
```

Comment:
This will not work. It just calls os.rename. I read the code and the docs and also tried it :)

Reply:
OK, then the docs of `Path.replace()` are wrong for saying "If _target_ points to an existing file **or directory**, it will be unconditionally replaced."

Anyway, thanks for trying.